lib/jxl/jxl_test.cc

```diff
@@ -1713,7 +1713,7 @@ TEST(JxlTest, RoundtripProgressive) {
   cparams.responsive = true;
   cparams.progressive_mode = true;
   CodecInOut io2;
-  EXPECT_LE(Roundtrip(&io, cparams, {}, &pool, &io2), 61600u);
+  EXPECT_LE(Roundtrip(&io, cparams, {}, &pool, &io2), 61610u);
```

Comment:
We should probably make this limit a bit larger :)

Reply:
done